rest/client/request.go

```diff
@@ -49,6 +49,10 @@ const (
 
 var HTTPConflictError = errors.New(evergreen.TaskConflict)
 
+// AuthError is a special error when the CLI receives 401 Unauthorized to
+// suggest logging in again as a possible solution to the error.
+var AuthError = errors.New("401 Unauthorized: User credentials are likely expired, try logging into the website again.")
```

Comment:
Could we be more explicit about the last part? Something like "try logging in via the Evergreen web UI"

Reply:
Done.